### DISCUSSION TITLE
Help about google drive index

### DISCUSSION THREAD

**Discussion by KuroProjects** [spencerwooo/onedrive-vercel-index#529] [Ideas]
well that's a silly thing to ask you
but is there any way you can do something so we can make a google drive index too
there are alot of indexes on internet but your UI is awesome and kind of amazing

this would be the best index if it works for google drive too
please share your views about this

**Comment by spencerwooo** [comment]
Probably won't be supporting drives other than OneDrive for the time being.

**Reply by KuroProjects** [reply]
oh okay
Great Work btw
